docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -0,0 +1,172 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+[[open-in-colab]] 
+
+# Inference with PEFT
+
+From the 🤗 PEFT [documentation](https://huggingface.co/docs/peft/index):
+
+> 🤗 PEFT, or Parameter-Efficient Fine-Tuning (PEFT), is a library for efficiently adapting pre-trained language models (PLMs) to various downstream applications without fine-tuning all the model's parameters. PEFT methods only fine-tune a small number of (extra) model parameters, significantly decreasing computational and storage costs because fine-tuning large-scale PLMs is prohibitively costly. Recent state-of-the-art PEFT techniques achieve performance comparable to that of full fine-tuning.
+
+But guess what? PEFT is not limited to just language models. It is modality-agnostic. This means it can be applied to pure vision models, vision-language models, speech models, and so on.
+
+🤗 PEFT is natively integrated with 🤗 Diffusers, enabling efficient multi-adapter inference. In this guide, you'll learn how to use different adapters with [Stable Diffusion XL (SDXL)](./pipelines/stable_diffusion/stable_diffusion_xl) for inference.
+
+Throughout this guide, you'll use LoRA as the main adapter technique, so we'll use the terms LoRA and adapter interchangeably. You should have some familiarity with LoRA, and if you don't, we welcome you to check out the [LoRA guide](https://huggingface.co/docs/peft/conceptual_guides/lora).
+
+Let's first install all the required libraries.
+
+```bash
+!pip install -q transformers accelerate
+!pip install -q git+https://github.com/huggingface/peft.git
+# To remove the branch once it's merged.
```

Comment:
Small tip, I would always use the template like `# TODO` for this type of comment, which makes it much easier to grep through the code base for things that need changing. In this case, I think `# FIXME` could be even more appropriate, in case this needs to be changed _before_ the PR is merged (`# TODO` would be something that needs changing eventually, but can still be merged as is).

Reply:
Thank you for the suggestions. These are not standardized across HF so I guess it's okay.